catalogue/src/main/java/org/openbaton/catalogue/security/User.java

```diff
@@ -17,31 +17,71 @@
 
 package org.openbaton.catalogue.security;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.Collection;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Table;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
 
 @Entity
 @Table(name = "USERS")
-public class User extends BaseUser {
+@JsonIgnoreProperties(
+  value = {"accountNonExpired", "accountNonLocked", "credentialsNonExpired"}
+) // otherwise there are problems with the Json SchemaValidator
+public class User extends BaseUser implements UserDetails {
 
   @Column(unique = true)
   private String username;
 
   private String password;
 
-  private Boolean enabled;
+  private boolean enabled;
```

Comment:
Is there a reason to use the type `boolean` instead of `Boolean`? for json schema validation we need `Boolean`. if there is no special reason for that i would keep `Boolean` with a default value

Reply:
I see it comes from UserDetails. Please be sure that create user/ update user still works